### ISSUE TITLE
Web - Search Results Page - Improper Link Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#89]
Each search result has a link that reads "Continue reading." which visually is the same for users. There were ten of them when I searched for the term "test". There is an added aria-label with the name of the page, but it might not be read by users with older machines or older versions of a screen reader that don't understand ARIA.

Search Results - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/search

**Comment by KyleLivingstonDOL** [comment]
Thanks John. When you state "older" machines/software, how old? Unfortunately we cannot code for every single version out there. We can try our best to code for as many universal readers as we can.

John, Can you please explain further why having 10 "continue reading" links that are visually the same is a 508 issue? I don't understand,but am not sure how we can make these different for 508 compliance.

**Comment by jfc3-dol** [comment]
The reason the link text "Continue reading" is an issue is because the text doesn't tell the user what they are going to continue to read. With a screen reader, the user can take the links out of content and read all of them without any other text near them. So when you have ten links that read "continue reading link, continue reading link, continue reading link" etc. they don't know what they are going to read. Even if read after it's confusing because the page name is read then another 20 or 30 words followed by the link "continue reading".

The means you give the user the extra information with the link depends on your audience meaning, if people are coming on old computers and browsers then you want to code it one way and if they are all new machine/devices, then it's different.

Please see attached screenshot showing what I searched for an then the "Continue reading" links.

![comply chain - web - multiple cintue reading links](https://user-images.githubusercontent.com/10423487/61236252-859f5e80-a705-11e9-9533-336675b4d01a.png)

**Comment by trevitalgit** [comment]
In my testing with VoiceOver, when I tab to “continue reading of step 2”, it announces “Link Continue Reading Step 2 Access Risks and Impacts - Why Conduct Risk and Impact Assessments” It looks fine to me testing with VoiceOver. Please let me know which old computers/browsers I should test against?

**Comment by jfc3-dol** [comment]
Thanks for correcting the issue.

My screen reader JAWS is now reading the page TITLE such as "Step 5: Monitor Compliance - Audit Tools" as part of the link after first reading "Link Continue reading."

I am going to close this issue.
